packages/edit-site/src/components/global-styles/preview-iframe.js

```diff
@@ -38,6 +38,7 @@ export default function PreviewIframe( {
 	label,
 	isFocused,
 	withHoverView,
+	previewStyles,
```

Comment:
At first I thought it was odd that a component named `PreviewIframe` didn't already receive `previewStyles` before this PR. Then I noticed that this is used to imperatively set CSS on the iframe. I am not a fan of this, although it works.

Why not tell the component what we're previewing and let it internally decide how to style itself? This way we also avoid magic values coming from other components.

Reply:
That's the approach in https://github.com/WordPress/gutenberg/pull/61280

Reply:
I agree with @draganescu, and I don't like the boolean `addGlobalStyles` prop from #61280.

We have a [`useGlobalStylesOutputWithConfig`](https://github.com/WordPress/gutenberg/blob/851e2a051b5c7a14669b83b83c1f9bd4b1e0bd86/packages/block-editor/src/components/global-styles/use-global-styles-output.js#L1311) function that allows you to pass exactly what is needed for styles. Or you could move the change up to the [`Variation`](https://github.com/WordPress/gutenberg/blob/962bd063bd6c75662b5facf5bc3408549816af93/packages/edit-site/src/components/global-styles/variations/variation.js#L25) component which provides the [`GlobalStylesContext`](https://github.com/WordPress/gutenberg/blob/962bd063bd6c75662b5facf5bc3408549816af93/packages/edit-site/src/components/global-styles/variations/variation.js#L28-L40).

It seems like a simpler solution to just change the source of the styles rather than overriding them with a new prop at this level.